arch/csr/Smcsrind/mireg.yaml

```diff
@@ -0,0 +1,30 @@
+---
+# yaml-language-server: $schema=../../../schemas/csr_schema.json
+$schema: csr_schema.json#
+kind: csr
+name: mireg
+long_name: Machine Indirect Register Alias 1
```

Comment:
I certainly understand why the "1" was added, but let's hew to the ratified text in the spec, which does not have the "1".